crates/evm/execution-errors/Cargo.toml

```diff
@@ -24,7 +24,6 @@ nybbles.workspace = true
 
 derive_more.workspace = true
 
-
 [features]
 default = ["std"]
-std = []
\ No newline at end of file
+std = ["reth-consensus/std"]
```

Comment:
What affect does this have overall?

Reply:
enables the `std` feature of `reth-consensus`, otherwise this errors
```console
error[E0277]: the trait bound `ConsensusError: StdError` is not satisfied
   --> crates/evm/execution-errors/src/lib.rs:209:66
    |
209 |             Self::Consensus(source) => std::error::Error::source(source),
    |                                        ------------------------- ^^^^^^ the trait `StdError` is not implemented for `ConsensusError`
    |                                        |
    |                                        required by a bound introduced by this call

For more information about this error, try `rustc --explain E0277`.
error: could not compile `reth-execution-errors` (lib) due to 1 previous error
```

Reply:
I ran `cargo c -p reth-execution-errors --no-default-features`. Changes in `reth-consensus` affected other crates, such as `reth-execution-errors`

Reply:
ohhhhhhhhh ok I see